.editorconfig

```diff
@@ -1,6 +1,6 @@
 [*.ts]
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
What's the plan regarding the indentation? Ultimately I think we should make it consistent everywhere, I guess this is just some intermediate step.

Reply:
Yeah, I noticed that .ts files have indentation set to 4, so I thought I'd change it to 2 to make it consistent, but it caused reformatting of a lot of files. I don't want to do it in KODI to prevent merge issues with KBSS TermIt, so I will probably do it first in KBSS TermIt and then merge it to KODI.